libs/order/src/reducers/order/order-reducer.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface DaffOrderReducerState {
```

Comment:
The point of this state is just to keep track of the loading state and errors that arise from order list and load.

Reply:
This doesn't seem terribly useful at the moment, I'd just remove it.

Reply:
Lets say a customer wants to view their recent orders. Knowing the loading state of these actions would allow the app dev to implement a ghost state or show a spinner to the customer. Its something that we have in pretty much all other state modules. I think we should keep it but I'll remove it if you really don't like it.

Reply:
I'm going to vote we keep it because we'll need it for the customer's profile in the future anyway.